nixos/modules/virtualisation/proxmox-image.nix

```diff
@@ -99,6 +106,16 @@ with lib;
         Additional options appended to qemu-server.conf
       '';
     };
+    diskPartitioning = mkOption {
```

Comment:
Why not just call this `partitionTableType` and drop `partitionTableType = cfg.diskPartitioning;` at line 142?